example/package.json

```diff
@@ -6,15 +6,15 @@
   "dependencies": {
     "fontfaceobserver": "^2.1.0",
     "prop-types": "^15.7.2",
-    "react": "link:../node_modules/react",
-    "react-dom": "^17.0.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
     "react-live": "^2.2.3",
     "react-scripts": "^4.0.0",
-    "react-showtime": "link:..",
+    "react-showtime": "https://github.com/azavea/react-showtime#feature/cek/netlify",
     "theme-ui": "^0.3.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "craco start",
```

Comment:
See [the comment above](https://github.com/azavea/react-showtime/pull/6/files#r526227945). This is only run during local development and doesn't impact Netlify.